src/main/java/com/dropbox/core/android/Auth.java

```diff
@@ -41,12 +53,17 @@ public static void startOAuth2Authentication(Context context, String appKey) {
      *                          user accounts. (note that user may still authorize the accounts).
      * @param sessionId         The SESSION_ID Extra on an OpenWith intent. null if dAuth
      *                          is being launched outside of OpenWith flow
+     * @param webHost           Server host used for oauth
      * @throws IllegalStateException if you have not correctly set up the AuthActivity in your
      *                               manifest, meaning that the Dropbox app will
      *                               not be able to redirect back to your app after auth.
      */
-    public static void startOAuth2Authentication(Context context, String appKey, String desiredUid,
-                                                 String[] alreadyAuthedUids, String sessionId) {
+    public static void startOAuth2Authentication(Context context,
+                                                 String appKey,
+                                                 String desiredUid,
+                                                 String[] alreadyAuthedUids,
+                                                 String sessionId,
+                                                 String webHost) {
```

Comment:
We can just add a new function and don't break existing public function signiture.

Reply:
this shouldn't break the existing public signature; it just overloads the method with another. The original signature is unchanged in the method above